### PR TITLE
Pager docs

### PR DESCRIPTION
**Pull request by svdimitr**

### CODE REVIEW

File: components/pager/overview.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+tags: telerik,blazor,pager,paging
+published: True
+position: 20
+---
+
+# Pager Overview
+
+The `Pager` component will enable you to add page navigation in your Blazor application. It follows the design patter of our components but as standalone one can be used outside of Grid.
+
+To use Telerik Pager component for Blazor:
```
Comment (marin-bratanov) [inline components/pager/overview.md]: These instructions need to be shorter, drop the "optional" things. Add instructions when and where to provide the new set of data, and add a loop to render some items (even if it is just a few `<div>` elements with the ID of the item). This is the most important explanation on how to use the component and it needs to offer this information.

File: components/pager/overview.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+}
+````
+
+>caption The result from the code snippet above
+
+![basic configuration of the pager](images/pager-basic-configuration-screenshot.jpg)
+
+## Features
+* `Class` - The CSS class that will be rendered on the main wrapping element of the Pager.
+* `Total` - **int** - Represents the total count of items in the pager.
```
Comment (marin-bratanov) [inline components/pager/overview.md]: The types of the parameters must also be in code blocks, not bold.

File: components/pager/overview.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+* `ButtonCount` - **int** - The number of pages to be visible. To take effect the `ButtonCount` must be **less** than the pages count (ButtonCount < Total / number of items on the page)
```
Comment (marin-bratanov) [inline components/pager/overview.md]: maybe this is the "maximum number of pages to be visible". Confirm with dev.

File: components/pager/overview.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+* `Page` and `@bind-Page` - **int** - Represents the current page of the pager. Those parameters are respectively for one and two-way data binding. If no `Page` or `@bind-Page` are provided they will default to the first page (1).
+* `PageChanged` - Fires when a new page is selected (used in one-way data binding).
+* `PageSize` - **int** - The number of items to be presented on a pages.
+
+## Examples
```
Comment (marin-bratanov) [inline components/pager/overview.md]: The first example should be something similar, but also include a `.Skip().Take()` call from an AllData collection into an CurrentPageData collection that you can use for the loop.

A second example here can call an `async` method that will return data based on the given PageSize and Page parameters so you can simulate load-on-demand from a service with it. Title the example accordingly.

File: components/pager/overview.md [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+
+>caption Observe the behavior of the Pager with two-way data binding
+
+````CSHTML
+@*This example showcases the usage of Page and PageChanged in conjunction*@
+
+<TelerikPager Total="TotalItems"
+              ButtonCount="ButtonCount"
+              PageSize="ItemsOnPage"
+              @bind-Page="CurrentPage">
```
Comment (marin-bratanov) [inline components/pager/overview.md]: For two-way binding the example should show not only how the component puts the page in an external variable, but also how changing the variable affects the pager. For example, bind it to a NumericTextbox as well so people can see how the pager reacts to that as well.

**Comment by marin-bratanov** [comment]
There must be a separate Events article.

File: components/pager/overview.md (hunk 1 above)
Comment (marin-bratanov) [reply components/pager/overview.md]: The Localization demo is a good template for this

File: components/pager/overview.md (hunk 2 above)
Comment (svdimitr) [reply components/pager/overview.md:50]: Done

File: components/pager/overview.md (hunk 1 above)
Comment (svdimitr) [reply components/pager/overview.md:15]: Added demo

File: components/pager/overview.md (hunk 5 above)
Comment (svdimitr) [reply components/pager/overview.md:99]: Done

File: components/pager/overview.md (hunk 3 above)
Comment (svdimitr) [reply components/pager/overview.md:51]: Done

**Pull request merged**